### DISCUSSION TITLE
Make the track disappear

### DISCUSSION THREAD

**Discussion by ABuffaloHerd** [Drewol/unnamed-sdvx-clone#529] [Q&A] [answered]
Is there a feature or a planned feature that makes the underlying track disappear so that only the notes and lasers are visible?

**Comment by Drewol** [comment, marked as answer]
you can add a line like this to your ksh file in a text editor `lane_toggle=192` and the lane will disappear at that time. The number decides how long it should take for the lane to completely disappear.

**Reply by ABuffaloHerd** [reply]
Thanks very much
